cmake/cfe-usersguide.doxyfile.in

```diff
@@ -17,11 +18,13 @@ GENERATE_LATEX         = YES
 STRIP_FROM_PATH       += @MISSION_SOURCE_DIR@/cfe/cmake/sample_defs
 INPUT                 += @MISSION_SOURCE_DIR@/cfe/cmake/sample_defs
 
-#PREDEFINED
 PREDEFINED            += @USERGUIDE_PREDEFINED@
 
 # Bring in the cFE header files for the documentation of the various API calls
 INPUT                 += @MISSION_USERGUIDE_HEADERFILES@
 
 # Resolve references
 INPUT                 += @MISSION_SOURCE_DIR@/psp/fsw/inc
+
+# List of target system-specific inputs
+@TGTSYSTEM_DOXYFILE_USER_CONTENT@
```

Comment:
Do we need psp/fsw/${CFE_SYSTEM_PSPNAME} included for the usersguide?  Seems like psp/fsw/inc is sufficient (covers APIs).  I'd only expect the PSPNAME dir to be in detaildesign.

Reply:
Yeah, I tend to agree.  The PSPNAME had been in the usersguide for some reason, I kept it in there, but probably should be removed.  As far as the detaildesign goes, it is already including the "psp" top level dir, which recursively picks up all of them, so this is redundant.

I will remove this from usersguide and see what happens.

Reply:
Version in d89c7ce7 removes PSPNAME from this